modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -0,0 +1,288 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-cluster-using-defaults-ocm_{context}"]
+= Creating a cluster with the default options using {cluster-manager}
+
+When using {cluster-manager-first} to create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can select the default options to create the cluster quickly.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
++
+[NOTE]
+====
+To successfully install ROSA 4.10 clusters, use the latest version of the ROSA CLI.
+====
+* You have logged in to your Red Hat account by using the `rosa` CLI.
+* You have verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
+
+.Procedure
+
+. Navigate to {cluster-manager-url} and select *Create cluster*.
+
+. On the *Create an OpenShift cluster* page, select *Create cluster* in the *{product-title}* row.
+
+. Review and complete the *Prerequisites* listed on the *Accounts and roles* page. Select the checkbox to acknowledge that you have read and completed all of the prerequisites.
+
+. Select an AWS account from the *Associated AWS account* drop-down menu. If no associated AWS accounts are found, click *Associate AWS account* and follow these steps:
+.. On the *Authenticate* page, click the copy button next to the `rosa login` command. The provided command includes your ROSA API login token.
+.. Run the copied command in the CLI to log in to your ROSA account:
++
+[source,terminal]
+----
+$ rosa login --token=<api_login_token> <1>
+----
+<1> Replace `<api_login_token>` with the token that is provided in the copied command.
++
+.Example output
+[source,terminal]
+----
+I: Logged in as '<username>' on 'https://api.openshift.com'
+----
+.. On the *Authenticate page* in {cluster-manager}, click *Next*.
+.. On the *OCM role* page, click the copy button next to the *Admin OCM role* command. The admin role enables automatic deployment of the cluster-specific Operator roles and the OpenID Connect (OIDC) provider by using {cluster-manager}.
+.. Run the copied command in the CLI and follow the prompts to create the {cluster-manager} role.
++
+The following example creates an admin {cluster-manager} user using the default options and `auto` mode for immediate STS resource creation. The example also links the {cluster-manager} role to a Red Hat organization:
```

Comment:
Suggestion:
The following example creates an admin `ocm-role` using the default options and `auto` mode for immediate STS resource creation. The example also links the `ocm-role` to a Red Hat organization:

Reply:
In my view, "`ocm-role`" is the `rosa create` subcommand. I can't see the role itself named `ocm-role` in any of the CLI outputs. The IAM role name once created also does not use that syntax. Also, having `ocm-role` in back-ticks implies that it is a reference to a code element. Please let me know if you have any feedback about this.

I did notice just now that the line incorrectly states "creates an admin OpenShift Cluster Manager user", where I meant to say ""creates an admin OpenShift Cluster Manager IAM role", so I have updated it accordingly.

Reply:
For further discussion on this topic, see https://issues.redhat.com/browse/OSDOCS-4058.